src/content/ui-api/kubectl-gs/get-appcatalogs.md

```diff
@@ -17,7 +17,7 @@ user_questions:
 
 # `kubectl gs get appcatalogs`
 
-Like with all `get` commands in `kubectl`, this command can be used to get details on one item, an [AppCatalog]({{< relref "/ui-api/management-api/crd/appcatalogs.application.giantswarm.io.md" >}})
+Like with all `get` commands in `kubectl`, this command can be used to get details on one item, an [App Catalog]({{< relref "/ui-api/management-api/crd/appcatalogs.application.giantswarm.io.md" >}})
```

Comment:
I think here it should remain as it is, since it's the `kind` of the CRD

```suggestion
Like with all `get` commands in `kubectl`, this command can be used to get details on one item, an [AppCatalog]({{< relref "/ui-api/management-api/crd/appcatalogs.application.giantswarm.io.md" >}})
```

Reply:
True, it should match the kind. 👍